### PR TITLE
Getting started tutorial for Visual Overview

### PR DESCRIPTION
**Pull request by ToonCasteele**

### CODE REVIEW

File: user-guide/Basic_Functionality/Visio/tutorials/Getting_started_with_basic_shapes.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+1. Right-click the shape group and select *Data* > *Define Shape Data*.
+
+1. Click the *New* button to add shape data with label `Enabled` and value `false`.
+
+1. Right-click the shape group and select *Group* > *Open Group*.
+
+1. Right-click the ellipse shape and select *Data* > *Define Shape Data*.
+
+1. Click the *New* button to add shape data with label `Enabled` and value `true`.
```
Comment (MariekeGO) [inline user-guide/Basic_Functionality/Visio/tutorials/Getting_started_with_basic_shapes.md:113]: Could you explain what the purpose of this is? I think some users might feel a little lost without that info. Right now, you explain it all in the paragraph at the end, but it I think it would be better to give the info bit by bit whenever relevant.

File: user-guide/Basic_Functionality/Visio/tutorials/Getting_started_with_basic_shapes.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+1. Right-click the rectangle shape and select *Data* > *Define Shape Data*.
+
+1. Click the *New* button to add shape data with label `Options` and value `NoAlarmColorFill`.
+
+1. Make sure you have no shapes selected, click the "..." in the top-right corner of the Advanced Editing panel and select *Add pretty hover*.
```
Comment (MariekeGO) [inline user-guide/Basic_Functionality/Visio/tutorials/Getting_started_with_basic_shapes.md:119]: This panel is not displayed by default. Could you add how users can get to see this panel, or add a tip with a link to a place where they can find this info?

File: user-guide/Basic_Functionality/Visio/tutorials/Getting_started_with_basic_shapes.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+   ![Add pretty hover](~/user-guide/images/AddPrettyHover.gif)
+
+1. Save your Visio file.
+
+In DataMiner Cube, only your ellipse will now be colored with the alarm state of the element, and only that shape will be clickable. The pretty hover will ensure that the highlighted area upon hover of the ellipse follows the ellipse rather than its bounding rectangle.
```
Comment (MariekeGO) [inline user-guide/Basic_Functionality/Visio/tutorials/Getting_started_with_basic_shapes.md:125]: "The pretty hover will ensure that the highlighted area upon hover of the ellipse follows the ellipse rather than its bounding rectangle."
=> This sentence isn't very clear. Could you rephrase this?

**Review by MariekeGO — CHANGES_REQUESTED** [review]

**Review by EdithLansens — APPROVED** [review]
